server/src/test/java/org/opensearch/rest/BaseRestHandlerTests.java

```diff
@@ -383,4 +388,14 @@ public String getName() {
         }
     }
 
+    public void testReplaceRoutesMethod() throws Exception {
+        List<Route> routes = Arrays.asList(new RestHandler.Route(Method.GET, "/path/test"),
+            new RestHandler.Route(Method.GET, "/path2/test"));
```

Comment:
If there is the test case for different paths there must be a test case for different methods as well.

Reply:
Done